ambientweather2sqlite/daemon.py

```
@@ -66,10 +68,14 @@
-            insert_observation(database_path, live_data)
+            database.insert_observation(live_data)
             wait_for_next_update(period_seconds)
     except KeyboardInterrupt:
-        print(f"\nStopping... results saved to {database_path}")
+        print("\nStopping...")
         if server is not None:
             server.shutdown()
+        try:
+            database.close()
+        except Exception:
+            pass
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)